pkg/envoy/lds/connection_manager.go

```diff
@@ -77,6 +79,26 @@ func getHTTPConnectionManager(routeName string, cfg configurator.Configurator, h
 			filters = append(filters, statsFilter)
 		}
 		connManager.HttpFilters = append(filters, connManager.HttpFilters...)
+
```

Comment:
I'd drop a comment here regarding what the subsequent headers are.

Reply:
Added